src/crsAdapter/TomaAdapter.js

```diff
@@ -317,7 +317,7 @@ class TomaAdapter {
      * @param lineNumber number
      * @returns {object}
      */
-    mapHotelServiceFromXmlObjectToAdapterObject(xml, lineNumber) {
+    mapHotelServiceFromXmlObjectToAdapterObject(xml, lineNumber, serviceType) {
```

Comment:
you already have the serviceType in `xml['KindOfService.' + lineNumber]` - there is no need to set it separately

Reply:
sure, I will change that.